### ISSUE TITLE
Benachrichtigungsfeld vom Telefon leert sich nicht bei Stelz-Posts

### ISSUE THREAD

**Issue by RundesBalli** [mopsalarm/Pr0#218]
Wenn man eine Nachricht in der Inbox hat, dann bekommt man ja im Telefon-Herunterziehmenü (also dieses von oben, wo die ganzen Benachrichtigungen drin sind) eine kleine Vorschau auf die PN bzw den Kommentar.
Da bekommt man auch eine Vorschau vom Bild, wenn jemand den man Stelzt einen Upload tätigt.

Wenn man über "Menü->Nachrichten" die PN / den Kommentar liest, dann verschwindet die Benachrichtigung automatisch. Wenn man allerdings über Menü->Nachrichten->Stelz-Abos->Post auf den Post geht, dann bleibt die Benachrichtigung im Herunterziehmenü.

Ist nur was kosmetisches, aber ich dacht mir, ich melds mal.

**Comment by mopsalarm** [comment]
Ja, das ist etwas was auf meiner Liste steht, die entsprechende API ist auch schon da, ich muss mich nun mal dran setzen und das bauen.

**Comment by RundesBalli** [comment]
Ok, klasse!

**Comment by mopsalarm** [comment]
Mache ich nun wirklich bald!

**Comment by RundesBalli** [comment]
Die Benachrichtigung geht übrigens nur weg wenn man über die Benachrichtigung den Stelzpost sieht. Auch nicht, wenn man über "Menü"->"Stelz" auf die Posts geht.
